cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -412,8 +412,8 @@ subroutine evp (dt)
                          indxti      (:,iblk), indxtj      (:,iblk), & 
                          indxni      (:,iblk), indxnj      (:,iblk), & 
                          aiN       (:,:,iblk), nmass     (:,:,iblk), & 
-                         nmassdti  (:,:,iblk), fcorN_blk  (:,:,iblk), & 
-                         nmask     (:,:,iblk),                       & 
+                         nmassdti  (:,:,iblk), fcorN_blk  (:,:,iblk),& 
+                         nmask     (:,:,iblk), nmask     (:,:,iblk), &                       
```

Comment:
maybe the second mask in this line could be made into an optional variable

Reply:
I think that I can eliminate the second mask and differentiate the two mask in the call as it is done for dyn_prep2 on E and N grid